static/libs-widget-ext.js

```diff
@@ -328,10 +330,19 @@ LibsWidgetExt.prototype.startSearching = function () {
 
         if (library.description) {
             if (library.description.toLowerCase().includes(lcSearchtext)) {
-                this.addSearchResult(libId, library, searchResults);
+
```

Comment:
Code looks good, but I can't see how this is different to the previous behaviour: how does deferring the `addSearchResult` til after the loop differ from doing it as you go along?

Reply:
because name matches are added first, only when all the libraries have been matched by name, the description matches are added

Reply:
Ah got it; thanks!